src/containers/RegistrationContainer.js

```diff
@@ -95,6 +95,11 @@ const RegistrationContainer = props => {
     }
   }
 
+  const stepButtonActions = () => {
+    if (pages[step].last) props.updateUser({ profileReady: true })
+    profileCreationAction()
+  }
```

Comment:
Correction: profileready is set here. Now it redirected to profile, and after second time no more modals, yay! But I just remembered that I think we agreed in the last meeting that it should actually redirect to groups-page after login?